.gitignore

```diff
@@ -60,4 +60,9 @@ rpc/server.gen.go
 rpc/client.gen.go
 
 # Ignored generated files by build/config/main.go
-build/config/gen*.json
\ No newline at end of file
+build/config/gen*.json
+
+# Binary downloads and manipulations by helm
```

Comment:
This is not related to operator, but helm instead. Helm is being used here to download third-party Open Source helm charts for observability part, and it basically downloads zip files. I don't want us to commit binaries, so that is why this is included in gitignore.